pylint/checkers/variables.py

```diff
@@ -1604,6 +1598,12 @@ def _check_consumer(
         if use_outer_definition:
             return (VariableVisitConsumerAction.CONTINUE, None)
 
+        if not (
+            self._is_undefined_variable_enabled
+            or self._is_used_before_assignment_enabled
+        ):
+            return (VariableVisitConsumerAction.RETURN, found_nodes)
+
```

Comment:
Let's just nuke this in a follow up PR. It's only going to cause problems. I'm worried about similar problems with the `CONTINUE` that is still underneath this.

Reply:
I think we can remove this bit entirely in this PR then. 
As I do not know exactly what code you are referring to with the ``CONTINUE``, I did not change anything else.

Reply:
Sorry for not being clear -- I was referring to this on L1653:
https://github.com/PyCQA/pylint/blob/5e1926e77fb5956f3776f7a75b18e809bce56a62/pylint/checkers/variables.py#L1653

My thought was that a similar test case (whatever should be hitting that CONTINUE) might fail if we don't remove this bit altogether. 👍🏻